client/components/app.jsx

```diff
@@ -20,11 +35,26 @@ export default class App extends React.Component {
   }
 
   render() {
-    return this.state.isLoading
-      ? <h1>Testing connections...</h1>
-      : <div>
-        <Header />
-        <ProductList />
-      </div>;
+    const window = this.state.view.name;
+    if (window === 'catalog') {
```

Comment:
Good work! You can also handle this with a switch statement as well when the number of items to switch between gets larger.